src/Shared/SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure/Data/IRefundPaymentRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data
+{
+    public interface IRefundPaymentRepository
+    {
+        void AddMany(List<RefundPaymentEntity> refunds);
```

Comment:
This should be in a PaymentsRepository
Also remember that because this is shared we have 3 different payments schemas, so you'll need to make allowance for selecting it in the method definition

Reply:
Aren't we adding this to the Refunds.Payments table?

Reply:
A refund is just a negative payment. It shares everything with a payment, and due to the location of the repository (shared infrastructure) it makes sense to use it to AddMany payments (no matter what kind of payment they are)